mvrx/src/main/kotlin/com/airbnb/mvrx/MavericksExtensions.kt

```diff
@@ -204,7 +204,7 @@ inline fun <T, reified VM : MavericksViewModel<S>, reified S : MavericksState> T
     }
 
 /**
- * [fragmentViewModel] except scoped to the current Activity. Use this to share state between different Fragments.
+ * ViewModel scoped to the Activity.
```

Comment:
This isn't much clearer. This sounds like it would be describing `activityViewModel` when this is for view models created inside of the Activity class itself.

Reply:
What do you propose?